cmd/antrea-agent/options.go

```diff
@@ -436,6 +450,9 @@ func (o *Options) setK8sNodeDefaultOptions() {
 	if o.config.AntreaProxy.ProxyLoadBalancerIPs == nil {
 		o.config.AntreaProxy.ProxyLoadBalancerIPs = ptr.To(true)
 	}
+	if o.config.AntreaProxy.ServiceHealthCheckServerBindAddress == "" {
+		o.config.AntreaProxy.ServiceHealthCheckServerBindAddress = net.JoinHostPort(net.IPv4zero.String(), strconv.Itoa(apis.AntreaProxyHealthServerPort))
```

Comment:
nit: no need for the `strconv` import, you can just use `fmt.Sprint(apis.AntreaProxyHealthServerPort)`